src/renderer/uia/UiaRenderer.cpp

```diff
@@ -13,9 +13,12 @@ using namespace Microsoft::Console::Types;
 // Routine Description:
 // - Constructs a UIA engine for console text
 //   which primarily notifies automation clients of any activity
-UiaEngine::UiaEngine() noexcept:
-    _isPainting(false),
-    _isEnabled(false),
+UiaEngine::UiaEngine(IUiaEventDispatcher* dispatcher) :
+    _dispatcher{ THROW_HR_IF_NULL(E_INVALIDARG, dispatcher) },
+    _isPainting{ false },
+    _selectionChanged{ false },
+    _isEnabled{ true },
+    _prevSelection{},

```

Comment:
I never actually set this anywhere. I should probably set it in `InvalidateSelection()` before exiting.

Reply:
Was this resolved? Cause I was gonna post the same comment....